templates/ammos-cubs-cognito.template.yaml

```diff
@@ -87,3 +104,6 @@ Outputs:
   CognitoUserPoolProviderUrl:
     Description: URL for the cognito provider (Issuer attribute)
     Value: !GetAtt UserPool.ProviderURL
+  CognitoDefaultUserTempPassword:
+    Description: Temporary Password for the Cognito User created by CognitoDefaultUser
+    Value: !GetAtt   CognitoDefaultUser.TemporaryPassword
```

Comment:
Can you clean up the whitespace here and also add an output for the username?
Otherwise looks great

Reply:
Yep sure!